challenges/objects-arrays.js

```diff
@@ -79,9 +114,25 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-const lowerCase = [];
+let names = [];
+
+zooAnimals.forEach(function (animal) {
+    names.push(animal["animal_name"]);
+})
+
+const lowerCase = names.map(zooAnimal => zooAnimal.toLowerCase());
```

Comment:
Try using only one method for this exercise and the next. Use only forEach or map.